#
# Necessary to deliver man pages to man{1,3,5,7}openssl sub-directories.
#
# upstream: https://github.com/openssl/openssl/pull/9554
#
--- old/util/process_docs.pl.orig	2019-08-07 14:31:29.204394234 +0200
+++ new/util/process_docs.pl	2019-08-07 14:42:39.442369387 +0200
@@ -100,7 +100,7 @@
         my $generate = { man  => "pod2man --name=$name --section=$podinfo{section} --center=OpenSSL --release=$config{version} \"$podpath\"",
                          html => "pod2html \"--podroot=$options{sourcedir}\" --htmldir=$updir --podpath=man1:man3:man5:man7 \"--infile=$podpath\" \"--title=$podname\" --quiet"
                          } -> {$options{type}};
-        my $output_dir = catdir($options{destdir}, "man$podinfo{section}");
+        my $output_dir = catdir($options{destdir}, "man$podinfo{section}" . (defined($options{suffix}) and $options{type} eq 'man' ? $options{suffix} : ""));
         my $output_file = $podname . $suffix;
         my $output_path = catfile($output_dir, $output_file);
 
